Assets/Tests/InputSystem/Plugins/UITests.cs

```diff
@@ -4016,6 +4016,260 @@ public IEnumerator UI_DisplayIndexMatchesDisplayMultiplePointers()
+        //ClickBothButtonOnCanvasCameraMode
+        ChangeCanvasRenderMode(leftEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ChangeCanvasRenderMode(rightEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ClickButton(leftButton);
+        ClickButton(rightButton);
+
+        //ClickBothButtonOnCanvasOverlayMode&CameraMode
+        ChangeCanvasRenderMode(leftEyeCanvas, RenderMode.ScreenSpaceOverlay);
+        ChangeCanvasRenderMode(rightEyeCanvas, RenderMode.ScreenSpaceCamera);
+        ClickButton(leftButton);
+        ClickButton(rightButton);
+
+        GameObject.DestroyImmediate(allParent);
+    }
+
+    [UnityTest]
+    public IEnumerator DragImageTest()
+    {
+        var playAllParent = GameObject.Instantiate(Resources.Load("PlayAllParent"));
+
+        //var m_Canvas = new GameObject("Canvas").AddComponent<Canvas>();
+        //m_Canvas.renderMode = RenderMode.ScreenSpaceOverlay;
+        //m_Canvas.gameObject.AddComponent<GraphicRaycaster>();
+        var m_Canvas = GameObject.Find("Left Eye Canvas Screen Overlay").GetComponent<Canvas>();
+        m_Canvas.targetDisplay = 0; //set 1 on secondary screen
+
+        var m_Image = new GameObject("Image").AddComponent<Image>();
+        m_Image.gameObject.transform.SetParent(m_Canvas.transform);
+        RectTransform imageRectTransform = m_Image.GetComponent<RectTransform>();
+        imageRectTransform.sizeDelta = new Vector2(100f, 100f);
+        imageRectTransform.localPosition = Vector3.zero;
+
+        GameObject go = new GameObject("Event System");
+        var m_EventSystem = go.AddComponent<EventSystem>();
+        EventSystem.current.pixelDragThreshold = 1; 
+
+        var m_StandaloneInputModule = go.AddComponent<StandaloneInputModule>();
+        var m_FakeBaseInput = go.AddComponent<FakeBaseInput>();
+        m_FakeBaseInput.image = m_Image;
+        m_FakeBaseInput.MousePosition = m_Image.GetComponent<RectTransform>().position; 
+
+        // Override input with FakeBaseInput so we can send fake mouse/keyboards button presses and touches
+        m_StandaloneInputModule.inputOverride = m_FakeBaseInput;
+
+        DragCallbackCheck callbackCheck = m_Image.gameObject.AddComponent<DragCallbackCheck>();
+        // Setting required input.mousePresent to fake mouse presence
+        m_FakeBaseInput.MousePresent = true;
+        yield return null; //important!
+
+        // Left mouse button down simulation
+        m_FakeBaseInput.MouseButtonDown[0] = true;
+        yield return null; //important!
+
+        // Left mouse button down flag needs to reset in the next frame
+        m_FakeBaseInput.MouseButtonDown[0] = false; //important!
+        for (int i = 0; i < 1000; i++)
+            yield return null;
+
+        // Left mouse button up simulation
+        m_FakeBaseInput.MouseButtonUp[0] = true;
+        yield return null;
+
+        // Left mouse button up flag needs to reset in the next frame
+        m_FakeBaseInput.MouseButtonUp[0] = false;
+        yield return null;
```

Comment:
To use the mouse, you can use these methods:

```
     // Move Mouse on Display 1
        Set(mouse.displayIndex, targetDisplay);
        Set(mouse.position, scene.From640x480ToScreen(10, 10));

        // Click on Display 1 - Tests reuse of last used pointer
        PressAndRelease(mouse.leftButton, queueEventOnly: true);
        yield return null;
```

They also have separate `Press` and `Release` if you want to separate them out.